website/content/gateway/api_reference/resources/authorizations/authorizations.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "authorizations"
+date: 2022-04-13T12:37:22+02:00
+anchor: "authorizations"
+weight: 105
+---
+#### Authorizations
+To reserve money on a cardholder’s bank account you make a new authorization resource.
+```shell
+POST https://gateway.clearhaus.com/authorizations
+```
+Authorizations can be created using different payment methods: `card`, `applepay`, `mobilepayonline`, `moto`, `vipps`. Exactly one payment method must be used.
+
+##### Parameters
+{{% description_list %}}
+{{% description_term %}}amount {{% regex %}}(0|[1-9][0-9]{,8}){{% /regex %}} {{% /description_term %}}
+{{% description_details %}}Amount in minor units of given currency (e.g. cents if in Euro).
+{{% /description_details %}}
+
+{{% description_term %}}currency {{% regex %}}[A-Z]{3}{{% /regex %}}{{% /description_term %}}
+{{% description_details %}}[3-letter currency code](/currencies). (Some exponents differ from ISO 4217.)
+{{% /description_details %}}
+
+{{% description_term %}} credential\_on\_file {{% regex %}}(store|use){{% /regex %}}{{% /description_term %}}
+{{% description_details %}} Indicate if the payment credential (e.g. PAN and expiry) will be stored for future use where the payment credential is not provided by the cardholder but collected from (encrypted) storage.
+
+`store`: The payment credential will be stored; it may only be stored if the authorization is approved.
+
+`use`: The payment credential has already been stored and is now being used.
+
+Default:
+
+
+- `use`, if `initiator` is `merchant` (`store` is invalid),
+
+- `store` if the authorization is first-in-series.
+
```

Comment:
```suggestion
- `use`, if `initiator` is `merchant` (`store` is invalid),
- `store` if the authorization is first-in-series.
```

If we want it to be similar to lines 48-50 below :man_shrugging: I haven't tested if it makes a difference.

Reply:
I think the reason for this order. Is you need to `store` the card before you can `use` it